cpp_coveralls/__init__.py

```diff
@@ -84,7 +84,13 @@ def run():
 
     if not args.no_gcov:
         coverage.run_gcov(args)
+
     cov_report = coverage.collect(args)
+
+    if not cov_report.get("repo_token") and not cov_report.get("service_job_id"):
```

Comment:
Can you raise the `Error` after parsing argument? We don't need to run `gcov` before finding out one of these fields is missing.